docs/src/00-doc/03-create-reusable-components/03_developing_components.stories.mdx

```diff
@@ -37,7 +37,7 @@ All component-level tokens must reference alias and global tokens listed in the
 
 The name of a component-level token file should match the name of the component. For example, a component implemented in `Button.vue` should have a `Button.json` containing its tokens.
 
-The recommended naming scheme for individual tokens is `wikit-<ComponentName>-[prop-][state-]<css-property>` - increasing specificity from left to right. A token containing the background-color (CSS property) value of a hovered (state) progressive (prop) button (component name) should be called `wikit-Button-progressive-hover-background-color`. Default prop values and states can be omitted from the name. A border-radius token that applies to all button types in all states can simply be named `wikit-Button-border-radius`.
+The recommended naming scheme for individual tokens is `wikit-<ComponentName>-[prop-][element-][state-]<css-property>` - increasing specificity from left to right. The order depends on the context however; states should follow the element on which the interaction occurred. Example: a "hover" on the "menu-item" impacting the "menu-item" style yields `wikit-Dropdown-progressive-menu-item-hover-background-color` while a "hover" on the "Dropdown" component impacting the "menu-item" style yields `wikit-Dropdown-progressive-hover-menu-item-background-color`.
```

Comment:
```suggestion
The recommended naming scheme for individual tokens is `wikit-<ComponentName>-[prop-][element-][state-]<css-property>` - increasing specificity from left to right. The order depends on the context however; states should follow the element on which the interaction occurred. 

For example: 

A "hover" on the "menu-item" impacting the "menu-item" style yields 
<code>wikit-Dropdown-progressive-<strong>menu-item-hover</strong>-background-color</code>
  
While a "hover" on the "Dropdown" component impacting the "menu-item" style yields
<code>wikit-Dropdown-<strong>progressive-hover</strong>-menu-item-background-color</code>.
```

Reply:
Thanks, @itamargiv ! I agree that the description is a bit hard to follow. Unfortunately I lack the necessary skills to illustrate the example, but I like the changes you made to at least split apart the long sentence and highlight the important differences!